docs/GETTING_STARTED.md

```diff
@@ -1,54 +1,87 @@
 # Getting Started
 
-This document provides basic instructions for training and evaluation using **pycls**.
+**pycls** can be used as a [library](#library-usage) (e.g. import pretrained models) or as a [framework](#framework-usage) (e.g. modify for your needs). This document provides brief installation instructions and basic usage examples for both use cases.
 
-- For general information about **pycls**, please see [`README.md`](../README.md)
-- For installation instructions, please see [`INSTALL.md`](INSTALL.md)
+## Library Usage
 
-## Training Models
+Install the package:
 
-Training on CIFAR with 1 GPU: 
+```
+pip install pycls
+```
+
+Load a pretrained model:
 
 ```
-python tools/train_net.py \
-    --cfg configs/archive/cifar/resnet/R-56_nds_1gpu.yaml \
-    OUT_DIR /tmp
+model = pycls.models.regnetx('400MF', pretrained=True)
 ```
 
-Training on ImageNet with 1 GPU:
+Please see the [`MODEL_ZOO.md`](MODEL_ZOO.md) for the available pretrained models.
+
+## Framework Usage
+
+Clone the repository:
 
 ```
-python tools/train_net.py \
-    --cfg configs/archive/imagenet/resnet/R-50-1x64d_step_1gpu.yaml \
+git clone https://github.com/facebookresearch/pycls
+```
+
+Install dependencies:
+
+```
+pip install -r requirements.txt
+```
+
+Set up modules:
+
+```
+python setup.py develop --user
+```
+
+Please see [`DATA.md`](DATA.md) for the instructions on setting up datasets.
+
+### Evaluation
+
+RegNetX-400MF on ImageNet with 8 GPUs:
+
+```
+python tools/test_net.py \
+    --cfg configs/dds_baselines/regnetx/RegNetX-400MF_dds_8gpu.yaml \
+    TEST.WEIGHTS https://dl.fbaipublicfiles.com/pycls/dds_baselines/160905967/RegNetX-400MF_dds_8gpu.pyth \
     OUT_DIR /tmp
 ```
 
-Training on ImageNet with 2 GPUs:
+### Training
+
+RegNetX-400MF on ImageNet with 8 GPUs:
 
 ```
 python tools/train_net.py \
-    --cfg configs/archive/imagenet/resnet/R-50-1x64d_step_2gpu.yaml \
+    --cfg configs/dds_baselines/regnetx/RegNetX-400MF_dds_8gpu.yaml \
     OUT_DIR /tmp
 ```
 
-## Finetuning Models
+### Finetuning
 
-Finetuning on ImageNet with 1 GPU:
+RegNetX-400MF on ImageNet with 8 GPUs:
 
 ```
 python tools/train_net.py \
-    --cfg configs/archive/imagenet/resnet/R-50-1x64d_step_1gpu.yaml \
-    TRAIN.WEIGHTS /path/to/weights/file \
+    --cfg configs/dds_baselines/regnetx/RegNetX-400MF_dds_8gpu.yaml \
+    TRAIN.WEIGHTS https://dl.fbaipublicfiles.com/pycls/dds_baselines/160905967/RegNetX-400MF_dds_8gpu.pyth \
     OUT_DIR /tmp
 ```
 
-## Evaluating Models
+### Timing
 
-Evaluation on ImageNet with 1 GPU:
+RegNetX-400MF with 1 GPU:
 
 ```
-python tools/test_net.py \
-    --cfg configs/archive/imagenet/resnet/R-50-1x64d_step_1gpu.yaml \
-    TEST.WEIGHTS /path/to/weights/file \
-    OUT_DIR /tmp
+python tools/time_net.py
+    --cfg configs/dds_baselines/regnetx/RegNetX-400MF_dds_8gpu.yaml \
+    NUM_GPUS 1 \
+    TRAIN.BATCH_SIZE 64 \
+    TEST.BATCH_SIZE 64 \
+    PREC_TIME.WARMUP_ITER 5 \
+    PREC_TIME.NUM_ITER 50
```

Comment:
@ir413 The new time_net code works with multiple GPUs so we don't really need these additional overrides?

Reply:
I think the intention here was to provide a command to reproduce model zoo style timings for one of the models rather than to account for the timing code not working in the multi GPU setting